GoogleForCreators/web-stories-wp
# Stickers: Discovery POC

## Context
Just a minimal implementation of a sticker element, created follow up tickets to increase feature richness. Didn't put behind a feature flag, cus you have to manually alter the story schema to add a sticker right now, so doesn't seem user facing with current implementation.

## Summary
Adds:
- Sample sticker example
- base sticker element schema
- relevant element components for editor and story output

## Relevant Technical Choices
Started off pulling from the media elements, then just stripped everything back to be more like the shape element since it seemed like the simplest implementation to getting complex svgs from templates in the stories.

Just stored a sticker type in the schema and use that to reference an inline svg. Ended up being pretty simple

## To-do
Added these follow up tasks and discussion points as tickets:
https://github.com/google/web-stories-wp/issues/6586
https://github.com/google/web-stories-wp/issues/6585
https://github.com/google/web-stories-wp/issues/6584
https://github.com/google/web-stories-wp/issues/6582

## User-facing changes
None


## Testing Instructions
-> Open the editor
-> Add a square shape to your story
-> Press `cmnd + option + shift + j` to open the story schema edit dialog
-> find the square shape element you added
-> update the element to have these properties
```
...
"type": "sticker",
"sticker": {
  "type": "SampleSticker"
},
...
```
See that a frying pan sample sticker shows up in the editor




### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6407
